src/data/hooks/useTvm.ts

```diff
@@ -1,42 +1,43 @@
 import { useQuery } from "@tanstack/react-query"
-import { getTvm as getTvm_AAVE_V2_STABLE_CELLAR } from "data/actions/AAVE_V2_STABLE_CELLAR/getTvm"
+import { getTvm } from "data/actions/common/getTvm"
 import { CellarKey, ConfigProps } from "data/types"
 import { useGetCellarQuery } from "generated/subgraph"
-import { AaveV2CellarV2 } from "src/abi/types"
 import { useCreateContracts } from "./useCreateContracts"
 
 export const useTvm = (config: ConfigProps) => {
   const { cellarContract } = useCreateContracts(config)
 
   const [cellarResult] = useGetCellarQuery({
     variables: {
-      cellarAddress: config.id,
-      cellarString: config.id,
+      cellarAddress: config.id.toLowerCase(),
```

Comment:
should we just lowercase all the addresses in the config?

Reply:
make sense, will change that

Reply:
solved in https://github.com/strangelove-ventures/sommelier/pull/537